src/Umbraco.Web/Mvc/EnsurePublishedContentRequestAttribute.cs

```diff
@@ -103,7 +103,7 @@ protected virtual void ConfigurePublishedContentRequest(PublishedRequest request
         {
             if (_contentId.HasValue)
             {
-                var content = Umbraco.Content(_contentId);
+                var content = Umbraco.Content(_contentId.Value);
                 if (content == null)
                 {
                     throw new InvalidOperationException("Could not resolve content with id " + _contentId);
```

Comment:
@nzdev @Shazwazza should it use `_contentId.Value` in `InvalidOperationException` as well?

Reply:
That's not going to have any perf impact